packages/editor/src/browser/editor.ts

```diff
@@ -29,6 +29,7 @@ export const TextEditorProvider = Symbol('TextEditorProvider');
 export type TextEditorProvider = (uri: URI) => Promise<TextEditor>;
 
 export interface TextEditorDocument extends lsp.TextDocument, Saveable, Disposable {
+    findMatches(options: FindMatchesOptions): FindMatch[];
```

Comment:
Mark this as optional, with a jsdoc as `@since 1.7.0` and add an entry in the changelog to mention that a new method was added to this interface. This way we should be able to make it a required method sometime later.